source/code-snippets/usage-examples/deleteMany.js

```diff
@@ -20,13 +20,8 @@ async function run() {
     // Query for all movies with the title "Santa Claus"
     const query = { title: "Santa Claus" };
 
-    collection.deleteMany(query, options, function(err, r) {
-      if (err) {
-        console.log("Error: " + err.errmsg);
-      } else {
-        console.log("Deleted " + r.deletedCount + " documents");
-      }
-    });
+    const result = await collection.deleteMany(query);
```

Comment:
What happened to `options`? Looks like it was already gone in the previous code and we included it on accident?

Reply:
that is precisely what happened